frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardParentToMatchBranchAction.java

```diff
@@ -90,7 +89,7 @@ public void update(AnActionEvent anActionEvent) {
   public void actionPerformed(AnActionEvent anActionEvent) {
 
     var project = getProject(anActionEvent);
-    var gitRepository = getSelectedVcsRepository(anActionEvent);
+    var gitRepository = project.getService(VcsRootComboBoxProvider.class).getSelectedVcsRepository();
```

Comment:
Hmm would be nice if we could expose a more narrow interface... in fact I guess we just need what's provided by `IGitRepositorySelectionProvider`, right? Would `project.getService(IGitRepositorySelectionProvider.class)` work here as well? :thinking: